js/tests/karma.conf.js

```diff
@@ -19,6 +19,7 @@ module.exports = (config) => {
     // list of files / patterns to load in the browser
     files: [
       jqueryFile,
+      'js/tests/vendor/sinon.min.js',
```

Comment:
Maybe should I add `karma-sinon` instead of adding this file 🤔

Reply:
If we can achieve the same without the manually copied dependency, then go for it 👍

Reply:
We'll still need the local file to run our unit test without Karma

Reply:
Your call. I'm not very familiar with the karma config.

Also, I don't quite get this whole change, care to elaborate a bit?

Reply:
Sure 😃 QUnit possess just a few methods compare to newer Framework of unit testing (Karma, Mocha), they are a lot of new concepts (Mocks, Spies, Stubs) which provide a better way to test our methods, because for example you can be sure a method have been called, or fake user data, or change temporary the behavior of our methods.

See : http://sinonjs.org/#try-it-out